packages/composite-checkout/src/components/checkout-steps.js

```diff
@@ -31,15 +41,53 @@ export function Checkout( { children, className } ) {
 		);
 	}
 
-	return children;
+	return (
+		<ContainerUI className={ joinClasses( [ className, 'composite-checkout' ] ) }>
+			<CheckoutStepDataContext.Provider
+				value={ {
+					activeStepNumber,
+					stepCompleteStatus,
+					setActiveStepNumber,
+					setStepCompleteStatus,
+					setTotalSteps,
+				} }
+			>
+				<MainContentUI
+					className={ joinClasses( [ className, 'checkout__content' ] ) }
+					isLastStepActive={ isThereAnotherNumberedStep }
+				>
+					{ children }
+
+					<SubmittButtonWrapperUI isLastStepActive={ ! isThereAnotherNumberedStep }>
```

Comment:
Is there a reason why Submit is spelled with two 't's here?

Reply:
Haha, my own haste. I will fix that; thanks.